packages/store/docs/store-taxonomy.md

```diff
@@ -0,0 +1,67 @@
+# A Taxonomy of Stores
+
+Some dimensions on which our Stores differ. For each of these dimensions, in the headings below we indicate the dominant default choice with an asterisk (*). A given Store maker should document which non-default points in this design space it supports. The `makeScalarMapStore` function makes stores that default to all the default choices below, allowing only the non-default option to specify a schema, to create a schematized store.
+
+All stores are Far objects (a kind of remotable) that may be snapshotted to (or restored from) a corresponding pass-by-copy objects. MapStores snapshot into CopyMaps. SetStores snapshot into CopySets.
+
+# API Differences
+
+## SetStores vs MapStores*
+
+* SetStores store only keys (of type Key), where each key is key-unique, i.e., it is not keyEQ to any other key in the set.
+* MapStores store key value associations, where the keys are key-unique Keys, as with sets, and the associated values are Passable.
+
+## Weak vs Strong*
+
+* A weak store does not enable its contents to be iterated. Only with the key of a given entry will a weak store reveal anything about the entry. This sometimes enables gc of entries that the collector knows can never be mentioned. But, unlike JavaScript WeakSets and WeakMaps, a weak store *may* allow other values as keys, such as numbers, that can be synthesized at will. Entries indexed by such synthesizable keys can never be collected.
+* A strong store does enable iteration of its contents, and so must retain those contents as long as the strong store itself exists.
+
+# Storage and Representation Differences
+
+## Scalar Keys Only* vs Composite Keys Allowed
+
+* Scalars are primitives or remotables, those keys that can be compared without looking inside objects. This directly reflects the semantics of JavaScript's Maps and Sets, which only index by scalar keys.
+* Composite keys are things like copyArrays, copyRecords, copySets, or copyMaps, that compare by structural key equality.
+
+## Unstructured* vs Schematized
+
+* An unstructued store is restricted only by the representation limitations listed here. They keys must be Keys. The values must be Passable. Scalar stores allow only scalar keys. Persistent stores only allow contents that are well defined after vat death.
+* A schematized store is further restricted by a pattern provided when it was created to serve as its schema. A schematized store will only allow contents that match its schema pattern.
+
+## Long* vs Short Expected Lifetime (WeakStores only)
```

Comment:
This now seems like an API detail that should not appear in the orienting taxonomy. But it should still appear somewhere prominent, so developers will encounter it when using these stores.